globals.json

```diff
@@ -557,5 +562,158 @@
 		"YUI": false,
 		"Y": false,
 		"YUI_config": false
+	},
+	"mootools": {
+		"$": false,
+		"$$": false,
+		"Asset": false,
+		"Browser": false,
+		"Chain": false,
+		"Class": false,
+		"Color": false,
+		"Cookie": false,
+		"Core": false,
+		"Document": false,
+		"DomReady": false,
+		"DOMEvent": false,
+		"DOMReady": false,
+		"Drag": false,
+		"Element": false,
+		"Elements": false,
+		"Event": false,
+		"Events": false,
+		"Fx": false,
+		"Group": false,
+		"Hash": false,
+		"HtmlTable": false,
+		"IFrame": false,
+		"IframeShim": false,
+		"InputValidator": false,
+		"instanceOf": false,
+		"Keyboard": false,
+		"Locale": false,
+		"Mask": false,
+		"MooTools": false,
+		"Native": false,
+		"Options": false,
+		"OverText": false,
+		"Request": false,
+		"Scroller": false,
+		"Slick": false,
+		"Slider": false,
+		"Sortables": false,
+		"Spinner": false,
+		"Swiff": false,
+		"Tips": false,
+		"Type": false,
+		"typeOf": false,
+		"URI": false,
+		"Window": false
+	},
+	"shelljs": {
+		"target": false,
+		"echo": false,
+		"exit": false,
+		"cd": false,
+		"pwd": false,
+		"ls": false,
+		"find": false,
+		"cp": false,
+		"rm": false,
+		"mv": false,
+		"mkdir": false,
+		"test": false,
+		"cat": false,
+		"sed": false,
+		"grep": false,
+		"which": false,
+		"dirs": false,
+		"pushd": false,
+		"popd": false,
+		"env": false,
+		"exec": false,
+		"chmod": false,
+		"config": false,
+		"error": false,
+		"tempdir": false
+	},
+	"prototypejs": {
+		"$": false,
+		"$$": false,
+		"$A": false,
+		"$F": false,
+		"$H": false,
+		"$R": false,
+		"$break": false,
+		"$continue": false,
+		"$w": false,
+		"Abstract": false,
+		"Ajax": false,
+		"Class": false,
+		"Enumerable": false,
+		"Element": false,
+		"Event": false,
+		"Field": false,
+		"Form": false,
+		"Hash": false,
+		"Insertion": false,
+		"ObjectRange": false,
+		"PeriodicalExecuter": false,
+		"Position": false,
+		"Prototype": false,
+		"Selector": false,
+		"Template": false,
+		"Toggle": false,
+		"Try": false,
+		"Autocompleter": false,
+		"Builder": false,
+		"Control": false,
+		"Draggable": false,
+		"Draggables": false,
+		"Droppables": false,
+		"Effect": false,
+		"Sortable": false,
+		"SortableObserver": false,
+		"Sound": false,
+		"Scriptaculous": false
+	},
+	"browserify": {
```

Comment:
@nzakas they may intersect, but it is done for jshint compliance
https://github.com/jshint/jshint/blob/master/src/vars.js
related PR: https://github.com/eslint/eslint/pull/1690